traffic_ops/traffic_ops_golang/routing/middleware/wrappers_test.go

```diff
@@ -26,6 +26,8 @@ import (
 	"encoding/json"
 	"flag"
 	"fmt"
+	"github.com/lestrrat-go/jwx/jwa"
```

Comment:
This is not organized in the correct order.

Reply:
Rearranging